doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should have no crashes.
```

Comment:
If you say it like this, it goes without saying and not worth documenting. Can try to be more specific. e.g. Problems with the data file (e.g. file corrupted, incorrect format, big file, read-only file etc.) should not cause crashes